.lintstagedrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
Love this

Will go really nicely with https://github.com/alphagov/govuk-frontend/pull/2849 with `standard --fix` → `eslint --fix`

Think we should stick to the ESM route (versus CommonJS) and make this **.lintstagedrc.mjs**?

Reply:
I'll update now the switch to `eslint` is merged.

Regarding moving to ESM, though, I'm not 100% sure. Most our config files are CJS by the looks of it and it'd make more sense to me to have them all in the same format. I'm not strongly attached to it, though.

Reply:
I tend to use ESM by default now, especially with lots of packages going "ESM only"

No rush to switch just for this one config though